src/pages/VerifySettings/__tests__/VerifySettings.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { waitFor } from "@testing-library/react";
+import { render } from "../../../testing";
+import { VerifySettings } from "../VerifySettings";
+
+describe("VerifySettings Page", () => {
+  test("render", async () => {
+    const { getByText } = render(<VerifySettings />, { wrappers: { theme: true }});
+
+    await waitFor(() => {
+      const text = getByText(/Verify Settings/i);
+      expect(text).toBeInTheDocument();
```

Comment:
Alright so, In this test you're only seeing if verify Settings is inside the page, not actually seeing if the implementation works fine. Some API mocks would be required in order to see if this implementation works.